core/src/main/java/io/questdb/griffin/engine/ops/OperationDispatcher.java

```diff
@@ -77,9 +77,6 @@ public OperationFuture execute(
                     closeOnDone
             );
             return future;
-        } catch (Throwable th) {
```

Comment:
This catch is required to handle OOM exceptions correctly. I'll move it to an upper level, so that it's not a problem for PGWire and push to the `vi_oom_logging` branch.